nixos/modules/services/misc/jellyfin.nix

```diff
@@ -64,14 +64,16 @@ in
         AmbientCapabilities = "";
         CapabilityBoundingSet = "";
 
-        # ProtectClock= adds DeviceAllow=char-rtc r
-        DeviceAllow = "";
+        # allow graphic cards
+        DeviceAllow = [
+          "/dev/dri/*"
+        ];
+        SupplementaryGroups = [ "video" ];
```

Comment:
The wiki also suggests `render`. However I don't think nixos has such a group.

Reply:
On my system, `/dev/dri/render*` devices belongs to the render group (and `/dev/dri/card*` to the video group), so I think it is needed.

Reply:
Same

```
$ ls /dev/dri/render* -l
crw-rw-rw- 1 root render 226, 128 May 28 22:38 /dev/dri/renderD128
```